recipes/biorbd-casadi/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set libname = "biorbd" %}
+{% set name = "biorbd-casadi" %}
+{% set version = "1.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyomeca/{{ libname }}/archive/Release_{{ version }}.tar.gz
+  sha256: 7682fa0c34d6c9a4415c8293d12089845390516845334dbb5c4fcaedb60872ef
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py<36 or py>37]
+  rpaths:
+    - lib/
+    - lib/{{ libname }}/
```

As this provides a shared library you should probably be using run_exports:
```suggestion
    - lib/{{ libname }}/
  run_exports:
    - {{ pin_subpackage('biorbd-casadi', max_pin='x.x') }}
```
`max_pin` should be adjusted to match the expected ABI stability

---

Thanks! Did not know this tag!